quantus_sdk/lib/src/services/datetime_formatting_service.dart

```diff
@@ -66,10 +70,13 @@ class DatetimeFormattingService {
         .padLeft(2, '0');
 
     return FormattedDuration(
+      days: days != '00' ? days : null,
       hours: hours,
       minutes: minutes,
       seconds: seconds,
-      formatted: '$hours:$minutes:$seconds',
+      formatted: days != '00'
```

Comment:
this is quite hokey too... should not make this string compare for "has days" also not 2 times